cohere/client.py

```diff
@@ -494,3 +496,28 @@ def __request(self, endpoint, json=None) -> Any:
                 self.__print_warning_msg(response)
 
         return res
+
+    def create_cluster_job(
+        self,
+        embeddings_url: str,
+        threshold: Optional[float] = None,
+        min_cluster_size: Optional[int] = None,
```

Comment:
why `Optional`? do we have values set in the backend if these aren't specified??

Reply:
there are default in backend